stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,491 @@
+        storyDescription: 'The feed list item can be displayed without an image by removing the <code class="docs-code">fd-avatar</code> element from the container.'
+    }
+};
+
+export const linked = () => `
+    <div class="fd-feed-list" aria-label="Feed List Item with user link example">
+        <div class="fd-feed-list__body">
+            <div 
+                class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list--link fd-feed-list__thumb"  
+                style="background-image: url(http://lorempixel.com/460/620/people/7/)"
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe"
+            ></div>
+                 
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">
+                        <a href="#" class="fd-link fd-link--emphasized" tabindex="0">Joe Doe: </a> 
+                    </span>
+                    Suspendisse facilisis sed odio in mollis. Donec quis justo aliquam, porta justo a, accumsan lorem.
+                </p>
+                
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+        </div>
+    </div>
+`;
+
+linked.storyName = 'Linked';
+linked.parameters = {
+    docs: {
+        storyDescription: 'The feed list item can display linked usernames (and images). To display a linked username, add the <code class="docs-code">fd-link</code> class with the <code class="docs-code">fd-link--emphasized</code> modifier to the <code class="docs-code">fd-feed-list__name element.</code>'
+    }
+};
+
+export const showMore = () => `
+    <div class="fd-feed-list" aria-label="Feed List Item with show more text example">
+        <div class="fd-feed-list__body">
+            <div 
+                class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list__thumb" 
+                style="background-image: url(http://lorempixel.com/460/620/people/7/)" 
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe"
+            ></div>
+                 
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    <span>Maecenas convallis velit quis felis dictum, in ultrices quam faucibus. Morbi tempor eu elit eu consequat. Sed at lorem a ex consequat pharetra. Etiam convallis odio at vulputate venenatis. In sit amet pharetra urna. Ut nulla nisi, porta at ligula a, elementum ullamcorper lectus. Suspendisse blandit, risus nec vestibulum volutpat, nulla neque rhoncus dolor, vitae rutrum lectus neque nec ligula. Maecenas pulvinar, ligula nec fringilla volutpat, urna quam rutrum tellus, vi...
+                    <a class="fd-feed-list__more" tabindex="0">More</a>        

```

Comment:
The `MORE` and `LESS` links are standard blue color links as per spec.
![image](https://user-images.githubusercontent.com/43764832/96050211-16477b80-0e2e-11eb-9881-27a9c23d6216.png)